model/src/main/java/org/projectnessie/api/params/FetchOption.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.api.params;
+
+import javax.annotation.Nullable;
+
+public enum FetchOption {
+  MINIMAL,
+  ALL;
+
+  /**
+   * Will return <code>true</code> if the given <code>fetchOption</code> is {@link FetchOption#ALL},
```

Comment:
Nit: for future: the `{@code ...}` javadoc syntax is a bit easier to read than HTML `<code>`